snuba/migrations/snuba_migrations/outcomes/0005_outcomes_copy.py

```diff
@@ -0,0 +1,128 @@
+from typing import Sequence
```

Comment:
This is a copy of 0001 in outcomes, so that github would detect a migration change. It will be removed before merge